core/src/test/java/com/graphhopper/GraphHopperTest.java

```diff
@@ -614,7 +614,7 @@ public void testCustomModel() {
         final String customCar = "custom_car";
         final String emptyCar = "empty_car";
         CustomModel customModel = new CustomModel();
-        customModel.addToSpeed(Statement.If("road_class == TERTIARY", Statement.Op.MULTIPLY, 0.1));
```

Comment:
the problem is now that previously barriers hindered using the track and when we avoid tertiary roads by so much it is forced into using tracks. In general we should think about having a bit slower speed on tracks for car, but this is for another issue.